Sources/Interaction/Manipulators/KeyboardCameraManipulator/index.js

```diff
@@ -30,12 +30,35 @@ function vtkKeyboardCameraManipulator(publicAPI, model) {
 
   //--------------------------------------------------------------------------
 
+  publicAPI.resetMovementSpeed = () => {
+    // Reset the movement speed to be proportional to the longest length
+    // of the renderer's bounds.
+    const { renderer } = internal;
+
+    const bounds = renderer.computeVisiblePropBounds();
+    const lengths = [
+      Math.abs(bounds[1] - bounds[0]),
+      Math.abs(bounds[3] - bounds[2]),
+      Math.abs(bounds[5] - bounds[4]),
+    ];
+
+    // Just a number that seems to work okay for our examples...
+    const divisor = 500;
+    model.movementSpeed = Math.max(...lengths) / divisor;
```

Comment:
You might want to use BoundingBox::getMaxLength() instead.

Reply:
Done.